### PR TITLE
[vODLA]Add inference time

### PR DESCRIPTION
**Pull request by tianboh**
Add inference time for both release and debug version requested by zhiwei.

### CODE REVIEW

File: ODLA/platforms/vodla/CMakeLists.txt [hunk 1]
```diff
@@ -1,3 +1,7 @@
+add_definitions(-DTIMING)
+add_definitions(-DLOOP_CNT=1)
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -DDEBUG -g3")
```
Comment (weimingzha0) [inline ODLA/platforms/vodla/CMakeLists.txt:3]: no need for this. 
you can do "cmake -DCMAKE_BUILD_TYPE=DEBUG"

File: ODLA/platforms/vodla/CMakeLists.txt [hunk 2]
```diff
@@ -1,3 +1,7 @@
+add_definitions(-DTIMING)
+add_definitions(-DLOOP_CNT=1)
```
Comment (weimingzha0) [inline ODLA/platforms/vodla/CMakeLists.txt:2]: what is LOOP_CNT used? for testing?

File: ODLA/platforms/vodla/CMakeLists.txt (hunk 1 above)
Comment (tianboh) [reply ODLA/platforms/vodla/CMakeLists.txt:3]: Okay, deleted now.

File: ODLA/platforms/vodla/CMakeLists.txt (hunk 2 above)
Comment (tianboh) [reply ODLA/platforms/vodla/CMakeLists.txt:2]: LOOP_CNT is the number of vodh_infer called to evaluate the average inference time in remote container. LOOP_CNT should set to one in release version.

**Review by weimingzha0 — APPROVED** [review]

**Pull request merged**